README.md

```diff
@@ -27,7 +27,7 @@ Network Policies in a very efficient manner.
 
 ## Prerequisites
 
-Antrea has been tested with Kubernetes clusters running version 1.16 or later.
+Antrea has been tested with Kubernetes clusters running version 1.19 or later.
```

Comment:
Hi @hjiajing you misunderstood the requirement, you need to input a field like 'deprecated: true' to our old CRDs version. Please check [here](https://github.com/kubernetes/kubernetes/blob/3ec6a387955b1240ad6d795663513f1ee12ceaec/cluster/addons/volumesnapshots/crd/snapshot.storage.k8s.io_volumesnapshotclasses.yaml#L97)

Reply:
Thanks for the reminder, I will add these fields soon.